src/UKHO.ERPFacade.API/Controllers/WebhookController.cs

```diff
@@ -201,9 +220,25 @@ public virtual async Task<IActionResult> LicenceUpdatedPublishedEventReceived([F
             await _azureBlobEventWriter.UploadEvent(licenceUpdatedEventJson.ToString(), LicenceUpdatedContainerName, correlationId + '/' + LicenceUpdatedFileName);
 
             _logger.LogInformation(EventIds.UploadedLicenceUpdatedPublishedEventInAzureBlob.ToEventId(), "Licence updated  published event is uploaded in blob storage successfully.");
-           
-            return new OkObjectResult(StatusCodes.Status200OK);
 
+            string recordOfSalesXmlTemplatePath = Path.Combine(Environment.CurrentDirectory, RosXmlFilePath);
+
+            XmlDocument rosPayload = new();
+            rosPayload.Load(recordOfSalesXmlTemplatePath);
+
+            HttpResponseMessage response = await _sapClient.PostEventData(rosPayload, _sapConfig.Value.SapServiceOperationForRecordOfSale, _sapConfig.Value.SapUsernameForRecordOfSale, _sapConfig.Value.SapPasswordForRecordOfSale);
+
+            if (!response.IsSuccessStatusCode)
+            {
+                _logger.LogError(EventIds.ErrorOccuredInSap.ToEventId(), "An error occured while processing your request in SAP. | {StatusCode}", response.StatusCode);
```

Comment:
Please use new Event Id for Licence updated event in case of error occurred from SAP end

Reply:
New event Id "ErrorOccurredInSapForLicenceUpdatedPublishedEvent" added for Licence updated event.